File: 프로그래머스/lv1/42748. K번째수/K번째수.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        
        for(int a=0;a<commands.length;a++){
            int i = commands[a][0];
            int j = commands[a][1];
            int k = commands[a][2];
            
            int[] arr2 = new int[j-i+1];
            for(int b=i-1;b<j;b++){
                arr2[b-(i-1)]= array[b];
            }
            Arrays.sort(arr2);
            answer[a]=arr2[k-1];
            
        }
        
        return answer;
    }
}

File: 프로그래머스/lv1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    
    commands.map((c)=>{
        const list = array.slice(c[0]-1,c[1]);
        list.sort((a,b)=>a-b);
        answer.push(list[c[2]-1]);
    })
    return answer;
}